Src/Plugins/Test.FunctionalTests.AllPlugins/TestData/PlaintextSecrets/ExpectedOutputs/VariousPats.sarif

```diff
@@ -10,168 +10,90 @@
-                  "startColumn": 1,
-                  "endLine": 12,
-                  "endColumn": 73,
-                  "charOffset": 442,
-                  "charLength": 72,
-                  "snippet": {
-                    "text": "aDVseGVxa3o0enVqa3l2emc1ZW1rZWpnZWx4ajR4NjR0dnl6Y2xwcmVrdWxvYXdiYno1cQ=="
+              "locations": [
+                {
+                  "physicalLocation": {
+                    "artifactLocation": {
+                      "uri": "src/Plugins/Test.FunctionalTests.AllPlugins/TestData/PlaintextSecrets/Inputs/VariousPats.txt",
+                      "uriBaseId": "SRC_ROOT"
+                    }
                   }
                 }
-              }
-            }
-          ]
-        },
-        {
-          "ruleId": "SEC1001/AzureDevOpsPersonalAccessToken",
-          "ruleIndex": 0,
-          "level": "error",
-          "message": {
-            "id": "Default",
-            "arguments": [
-              "VariousPats.txt",
-              "plaintext",
-              "Azure DevOps personal access token"
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
-                  "uri": "src/Plugins/Test.FunctionalTests.AllPlugins/TestData/PlaintextSecrets/Inputs/VariousPats.txt",
-                  "uriBaseId": "SRC_ROOT"
-                },
-                "region": {
-                  "startLine": 3,
-                  "startColumn": 1,
-                  "endLine": 3,
-                  "endColumn": 53,
-                  "charOffset": 86,
-                  "charLength": 52,
-                  "snippet": {
-                    "text": "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdead"
-                  }
-                }
-              }
-            }
-          ]
-        },
-        {
-          "ruleId": "SEC1001/AzureDevOpsPersonalAccessToken",
-          "ruleIndex": 0,
-          "level": "error",
-          "message": {
-            "id": "Default",
-            "arguments": [
-              "VariousPats.txt",
-              "plaintext",
-              "Azure DevOps personal access token"
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
-                  "uri": "src/Plugins/Test.FunctionalTests.AllPlugins/TestData/PlaintextSecrets/Inputs/VariousPats.txt",
-                  "uriBaseId": "SRC_ROOT"
+              ],
+              "message": {
+                "text": "An exception of type 'ArgumentOutOfRangeException' was raised analyzing 'VariousPats.txt' for check 'SearchSkimmer' (which has been disabled). The exception may have resulted from a problem related to parsing the analysis target, and not specific to the rule, however."
+              },
+              "level": "error",
+              "exception": {
+                "kind": "ArgumentOutOfRangeException",
```

Comment:
>ArgumentOutOfRangeException [](start = 25, length = 27)

Thanks to us now honoring 'no match' properly, this SARIF log now has two false positive eliminated from it (two patterns that don't pass the PAT CRC check).